packages/core/src/components/forms/_controls.scss

```diff
@@ -46,10 +93,10 @@ $control-checked-background-color-active: nth(map-get($button-intents, "primary"
   }
 
   .#{$ns}-control-indicator {
-    position: absolute;
-    top: 0;
-    left: 0;
-    margin: 0;
+    display: inline-block;
```

Comment:
this seems pretty major

Reply:
it is, but it's also wholly contained inside the component. no API has been changed.